src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -104,6 +98,10 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER OR CLR_CMAKE_TARGET_WASI)
 
     set_target_properties(System.IO.Compression.Native-Static PROPERTIES OUTPUT_NAME System.IO.Compression.Native CLEAN_DIRECT_OUTPUT 1)
 else ()
+
+    # Disable C4334 warning for Zlib and Brotli
+    add_compile_options($<$<COMPILE_LANG_AND_ID:C,MSVC>:/wd4334>)
```

Comment:
For anyone interested, I fixed 4334 upstream for zlib-ng, and two other errors: https://github.com/zlib-ng/zlib-ng/pull/1762

If you're seeing the error in brotli, maybe consider submitting an upstream PR for them too?

Reply:
Is 4334 on the must-fix SDL list?

Reply:
Yes, it is on the must fix list. It's a level 3 warning and all level 3 warnings are on the must-fix list.

I'll prep a fix and push it here.

Reply:
Pushed a fix